supply/lib/supply/uploader.rb

```diff
@@ -110,6 +110,7 @@ def fetch_track_and_release!(track, version_code, status = nil)
       track = tracks.first
       releases = track.releases
 
+      releases = releases.reject { |r| r.version_codes.nil? || r.version_codes.empty? } if version_code
       releases = releases.select { |r| r.status == status } if status
       releases = releases.select { |r| r.version_codes.map(&:to_s).include?(version_code.to_s) } if version_code
```

Comment:
How about this, instead? It'd more optimized I think:
```suggestion
      releases = releases.select { |r| r.version_codes&.map(&:to_s).include?(version_code.to_s) } if version_code
```

Reply:
I tried this and it throws another error due to the `include?` method. `releases = releases.select { |r| r.version_codes&.map(&:to_s)&.include?(version_code.to_s) } if version_code` fixes the problem

Reply:
But these changes are not compatible with target ruby version. Rubocop complains:

`supply/lib/supply/uploader.rb:115:55: E: unexpected token error
(Using Ruby 2.0 parser; configure using TargetRubyVersion parameter, under AllCops)
      releases = releases.select { |r| r.version_codes&.map(&:to_s)&.include?(version_code.to_s) } if version_code

supply/lib/supply/uploader.rb:115:68: E: unexpected token error
(Using Ruby 2.0 parser; configure using TargetRubyVersion parameter, under AllCops)
      releases = releases.select { |r| r.version_codes&.map(&:to_s)&.include?(version_code.to_s) } if version_code`

Reply:
One of these days we'll be able to use all of the fun stuff 😊